renderdoc/driver/vulkan/vk_resources.cpp

```diff
@@ -706,6 +720,18 @@ uint32_t GetByteSize(uint32_t Width, uint32_t Height, uint32_t Depth, VkFormat F
     case VK_FORMAT_ASTC_12x12_SRGB_BLOCK:
       astc[0] = 12;
       astc[1] = 12;
+	  break;
+    case VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG:
+    case VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG:
+    case VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG:
+    case VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG:
+      ret = static_cast<uint32_t>(ceil(w/8.0) * ceil(h/4.0) * 8.0);
```

Comment:
Use plain casts `uint32_t(...);` or `(uint32_t)(...);` instead of `static_cast` please :smile:.